content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -0,0 +1,33 @@
+---
+title: kubeadm reset
+content_type: conceito
+weight: 60
+---
+<!-- overview -->
+Executa uma melhor reversão das alterações feitas pelo `kubeadm init` ou `kubeadm join`.
+
+<!-- body -->
+{{< include "generated/kubeadm_reset.md" >}}
+
+### Redefinindo o fluxo de trabalho {#reset-workflow}
+
+O `kubeadm reset` é o responsável por limpar um sistema de arquivos local dos nós a partir dos arquivos que foram criados usando os comandos `kubeadm init` ou `kubeadm join`. O `reset` dos nós do plano de controle também remove o etcd local do nó do cluster etcd.
```

Comment:
This sentence needs a little bit of rethinking. It's not immediately clear to me from reading this that `kubeadm reset` removes files generated by `init` or `join`.
It's better to use "camada de gerenciamento" instead of "plano de controle".